tests/impl.cpp

```diff
@@ -6968,7 +6968,22 @@ result_t test_mm_extract_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_extract_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    int64_t *_a = (int64_t *) impl.mTestIntPointer1;
+
+    __m128i a = do_mm_load_ps((const int32_t *) _a);
+    __int64_t c;
```

Comment:
You can even rewrite as following:
```c
__int64_t c = _mm_extract_epi64(a, i & 0x1);
```

Reply:
The provided implementation can't be compiled. The following error would occur

```
error: selector must be an integer constant in the range [0, 1]
```

Reply:
My bad. I forgot the qualifier.